### DISCUSSION TITLE
You must specify `level` when using `prediction_intervals

### DISCUSSION THREAD

**Discussion by deryansyah** [Nixtla/statsforecast#634] [Q&A]
Hello again,

I'm encountering this error when I tried to run this code

```
from statsforecast.models import (
    AutoRegressive,
    ARIMA,
    SeasonalNaive
)

model_arima = [
    AutoRegressive(lags=[12]),
    ARIMA(order=(1, 0, 0), season_length=12),
    SeasonalNaive
]

sf = StatsForecast(
    df=df, 
    models=model_arima,
    freq='MS', 
    n_jobs=1,
    fallback_model = SeasonalNaive(season_length=12),
    verbose=True
)

forecasts_df = sf.forecast(h=18, fitted=True, level=(90, 10))
forecasts_df.head()

```

and this is the error:

> During handling of the above exception, another exception occurred:
> 
> KeyError                                  Traceback (most recent call last)
> Cell In[15], line 1
> ----> 1 forecasts_df = sf.forecast(h=18, fitted=True, level=(90, 10))
>       2 forecasts_df.head()
> 
> File ~\AppData\Roaming\Python\Python311\site-packages\statsforecast\core.py:1899, in StatsForecast.forecast(self, h, df, X_df, level, fitted, sort_df, prediction_intervals)
>    1895     raise ValueError(
>    1896         "You must specify `level` when using `prediction_intervals`"
>    1897     )

Thank you

**Comment by jmoralez** [comment]
Hey. What version of statsforecast are you using?

**Reply by deryansyah** [reply]
Hi @jmoralez, my problem has been solved because some of unique_id have less than 12 data while I set the season_length=12.

but the `.forecast` mode (without fitted) can run smoothly for unique_id's that have less than 12 data for the season_length=12

btw I'm using statsforecast 1.6.0

**Reply by jmoralez** [reply]
If you have a reproducible example can you open an issue, please?
